Running in wrapper mode for '303'

Running python job.

[name: "CPU-QUOTA"
int_val: 35
int_val: 58
int_val: 90
int_val: 19
int_val: 14
int_val: 98
int_val: 21
, name: "DISK"
int_val: 8
int_val: 23
int_val: 14
int_val: 15
int_val: 20
int_val: 5
int_val: 11
, name: "MEMORY"
int_val: 5
int_val: 30
int_val: 7
int_val: 33
int_val: 10
int_val: 14
int_val: 21
, name: "NET"
int_val: 20
int_val: 8
int_val: 13
int_val: 25
int_val: 12
int_val: 26
int_val: 13
]
The paramters are {u'CPU-QUOTA': array([35, 58, 90, 19, 14, 98, 21]), u'NET': array([20,  8, 13, 25, 12, 26, 13]), u'DISK': array([ 8, 23, 14, 15, 20,  5, 11]), u'MEMORY': array([ 5, 30,  7, 33, 10, 14, 21])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 145000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 40960.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 19293798.4
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 1125000000.0
Recovering the memory utilization
Setting cpu quota to 87500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 102400.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 6710886.4
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 187500000.0
Recovering the memory utilization
Setting cpu quota to 225000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 66560.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 11744051.2
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 262500000.0
Recovering the memory utilization
Setting cpu quota to 87500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 102400.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 6710886.4
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 187500000.0
Recovering the memory utilization
Setting cpu quota to 225000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 66560.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 11744051.2
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 262500000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7f40384a7e60>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.37 seconds.
